atlasaction/action.go

```diff
@@ -596,28 +596,33 @@ func createRunContext(ctx context.Context, act *githubactions.Action) (*atlasexe
 	if err := envconfig.Process(ctx, &a); err != nil {
 		return nil, fmt.Errorf("failed to load actor: %w", err)
 	}
+	url := ev.PullRequest.URL
```

Comment:
this logic is taken from the old action:
![Uploading Screen Shot 2024-02-29 at 17.25.34.png…]()